index.d.ts

```diff
@@ -28,6 +28,7 @@ declare module "react-native-tesseract-ocr" {
   export const LANG_LATIN = "lat";
   export const LANG_LITHUANIAN = "lit";
   export const LANG_NEPALI = "nep";
+  export const LANG_NEDERLAND = "nld"
```

Comment:
Please add it to the `Lang` type union as well

https://github.com/jonathanpalma/react-native-tesseract-ocr/blob/73ad2cdc75ac524a2d1a0d6aec961119113f8eeb/index.d.ts#L50-L91